<launch>
    
    <!-- Setting the parameter for number of images to take -->
    <arg name="calib_image_num" default="8"/>
    <param name="calib_image_num" value="$(arg calib_image_num)" type="double"/> 

    <!-- start the calibration node -->
    <node pkg="lab4" type="lab4_calibration.py" name="calibration_node" output="screen"/>

    
</launch>